deployer/commands/generate/cryptnono_config/__init__.py

```diff
@@ -0,0 +1,34 @@
+import shutil
+import subprocess
+from pathlib import Path
+
+from deployer.cli_app import generate_app
+
+HERE = Path(__file__).parent
+REPO_ROOT_PATH = HERE.parent.parent.parent.parent
+
+
+@generate_app.command()
+def cryptnono_secret_config():
+    """
+    Update the secret blocklist for cryptnono
+    """
+    unencrypted_path = HERE / "unencrypted_secret_blocklist.py"
```

Comment:
This should be following the pattern on how to name encrypted / decrypted conten to make our .gitignore file protect us from adding it unencrypted.

```gitignore
# This section handles sensitive content that is or isn't encrypted in our repo.
#
# Ignore files (but not directories) with secret in their name
*secret*
!*secret*/
# Don't ignore enc- prefixed files
!enc-*
# Don't ignore Helm chart templates with secret in their name as
# they are not supposed to contain sensitive content but can often
# have secret in their name.
!**/templates/**/*secret*.yaml
```

Let's make `enc-<name>.secret.<file extension>` to `<name>.secret.<file extension>`.

```suggestion
    unencrypted_path = HERE / "blocklist-generator.secret.py"
```

Reply:
The paths already take into account our gitignore. The encrypted file to be checked in is called `enc-something.secret.py` so it can be checked in. Our gitignore ignores anything with secret but without enc, so this temporary unencrypted file *will* be ignored. We can't use dashes in there because then the python import statement would not work, and the same is true of additional dots in the filename. As an additional precaution, the `finally` block deletes this file as well.